src/config/dynamic-theme-fixes.config

```diff
@@ -350,6 +350,22 @@ CSS
     background-color: ${white} !important;
 }
 
+div.milestone.notReached .milestone-marker::before {
```

Comment:
```CSS
div.milestone.notReached .milestone-marker::before {
    border-color: var(--darkreader-neutral-text) !important;
}
```
Use 4 spaces and `!important` for every CSS declaration. Use this as well for the other ones.